interfaces/HO-Portal/src/app/programs/program-list/program-list.component.html

```diff
@@ -4,7 +4,7 @@
       <ion-menu-button></ion-menu-button>
     </ion-buttons>
     <ion-title>
-      {{ 'page.programs.PROGRAMS'|translate }}
+      {{ 'page.programs.program-list.title'|translate }}
```

Comment:
\o/ Yay! Way more specific and nice.

But the other page had `pageTitle` as the last element; Could that be the same everywhere?

Reply:
changed everywhere.